src/frontend/app.ts

```diff
@@ -304,10 +304,17 @@ export default (octokitApp: App): Express => {
 
 		statsd.increment(metricError.githubErrorRendered, tags);
 
-		return res.status(errorStatusCode).render("github-error.hbs", {
-			title: "GitHub + Jira integration",
-			nonce: res.locals.nonce
-		});
+		if (await booleanFlag(BooleanFlags.NEW_GITHUB_CONFIG_PAGE, false)) {
+			return res.status(errorStatusCode).render("github-error.hbs", {
```

Comment:
supernit looks like the only difference between if-body and else-body is the name of the template; could probably be simplified by having a ternary operator instead